Makefile

```
@@ -55,6 +55,21 @@ else
 	helm package manifests/$(MOD) --version $(VERSION) --app-version $(VERSION) --dependency-update --destination .artifacts
 endif
 
+.PHONY: build/plugin
+build/plugin: PLUGINS ?= "" # comma separated list of plugins. eg: PLUGINS=kubernetes,ecs,lambda
+build/plugin: PLUGINS_BIN_DIR ?= ~/.pipecd/plugins
```

IMO, the default of piped's pluginDir is `~/.piped/plugins`, so it's nice to make this the same as this.

https://github.com/pipe-cd/pipecd/blob/74de8a0b69fc34fe177c303b2aa6ad54c8e46841/pkg/app/pipedv1/cmd/piped/piped.go#L112

```suggestion
build/plugin: PLUGINS_BIN_DIR ?= ~/.piped/plugins
```

---

Nice catch 👍